### PR TITLE
Fixed refresh flickering and thumbnail reloading

### PR DESCRIPTION
**Pull request by brianytang**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @brianytang has signed the CLA. Thanks for the pull request!

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java [hunk 1]
```diff
@@ -391,7 +391,7 @@ public boolean onLongClick(View v) {
         public void onClick(View v) {
             if (mListener.getMultiSelectHandler() != null && mListener.getMultiSelectHandler().isEnabled()) {
                 mListener.getMultiSelectHandler().toggle(mItem);
-                onBindBoxItemViewHolder(this);
+                onBindBoxItemViewHolder(this, null);
```
Comment (doncung) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java]: should be mItem

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java [hunk 2]
```diff
@@ -246,32 +241,37 @@ public void bindItem(BoxItem item) {
...
-                String size = item.getSize() != null ?
-                        localFileSizeToDisplay(item.getSize()) :
+                String size = itemToBind.getSize() != null ?
+                        localFileSizeToDisplay(itemToBind.getSize()) :
                         "";
-                description = String.format(Locale.ENGLISH, "%s  • %s", modifiedAt, size);
-                mController.getThumbnailManager().loadThumbnail(item, holder.getThumbView());
+                String description = String.format(Locale.ENGLISH, "%s  • %s", modifiedAt, size);
+                holder.getMetaDescription().setText(description);
+                mController.getThumbnailManager().loadThumbnail(itemToBind, holder.getThumbView());
```
Comment (doncung) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java]: Can we modify the loadThumbnail method to search to see if it already has the icon downloaded?

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxSearchAdapter.java [hunk 3]
```diff
@@ -57,7 +55,7 @@ public SearchViewHolder(View itemView) {
         }
 
         @Override
-        protected void onBindBoxItemViewHolder(BoxItemViewHolder holder) {
+        protected void onBindBoxItemViewHolder(BoxItemViewHolder holder, BoxItem itemToBind) {
             if (holder.getItem() == null || holder.getItem() == null) {
```
Comment (doncung) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxSearchAdapter.java]: Do these need to check itemToBind?

**Pull request merged**